configuration/README.md

```diff
@@ -38,4 +38,5 @@ Some basic guides to configuring your Raspberry Pi.
     - How to configure screen blanking/screen saver
 - [The boot folder](boot_folder.md)
     - What it's for and what's in it
-
+- [Network File System](nfs.md)
+    - How to setup a NFS and connect clients to it
```

Comment:
Sentence doesn't make sense. I'm guessing it is means to read "how to set up an NFS server and connect clients to it".